src/elements/featured-videos.html

```diff
@@ -0,0 +1,330 @@
+        display: block;
+      }
+
+      .videos-wrapper {
+        padding: 0;
+      }
+
+      .header {
+        margin: 24px auto;
+        padding: 0 16px;
+      }
+
+      .title {
+        font-size: 24px;
+      }
+
+      .icon-label {
+        margin-left: 10px;
+      }
+
+      .video-icon {
+        margin-left: 0;
+        padding-left: 0;
+      }
+
+      .video-list {
+        position: relative;
+        margin: 0 auto;
+        max-width: 1280px;
+        height: 250px;
+        overflow: hidden;
+        margin-left: -15px;
+      }
+
+      .videos {
+        transition: all 0.4s cubic-bezier(0, 0, 0.2, 1);
+        will-change: transition;
+        backface-visibility: hidden;
+        transform: translateX(0px);
+      }
+
+      .video-item {
+        width: 315px;
+        min-width: 315px;
+        height: 213px;
+        box-sizing: border-box;
+        position: relative;
+        padding: 0 15px;
+        box-sizing: border-box;
+      }
+
+      .preview-image {
+        display: block;
+        cursor: pointer;
+        width: 280px;
+        min-width: 270px;
+        height: 177px;
+        border-radius: 5px;
+        margin-bottom: 10px;
+        overflow: hidden;
+        box-sizing: border-box;
+      }
+
+      .video-play {
+        position: absolute;
+        z-index: 1;
+        width: 60px;
+        height: 60px;
+        color: #fff;
+        opacity: 0.8;
+        left: 50%;
+        top: 46%;
+        transform: translate(-50%, -54%);
+      }
+
+      .video-title {
+        font-size: 18px;
+        line-height: 26px;
+        color: var(--secondary-text-color);
+      }
+      
+      @media (min-width: 768px) {    
+
+        .videos-wrapper {
+          padding: 60px 40px;
+        }
+
+        .header {
+          padding: 0;
+        }
+
+        .chevron-icon {
+          width: 100px;
+          height: 100px;
+          color: #FFF;
+          opacity: 0.6;
+          position: absolute;
+          z-index: 1;
+          top: 40%;
+          transform: translate(0, -55%);
+          display: none;
+          background-color: var(--default-primary-color);
+          border-radius: 50%;
+        }
+
+        .video-list:hover .chevron-icon {
+          display: block;
+        }
+
+        .next-video {
+          justify-content: flex-end;
+          right: -25px;
+        }
+
+        .last-video {
+          justify-content: flex-start;
+          left: -30px;
+        }
+      }
+
+      @media (min-width: 868px) {
+
+        .preview-image {
+          width: 100%;
+          min-width: 240px;
+          height: 221px;
+        }
+
+        .video-item {
+          width: 420px;
+          min-width: 420px;
+          height: 260px;
+        }
+
+         .video-list {
+          min-height: 300px;
+         }
+      }
+    </style>
+
+    <iron-media-query id="mq-phone" full query="(max-width: {$ mediaQueries.xs.max $})" query-matches="{{isPhoneSize}}"></iron-media-query>
```

Comment:
No need to define it. You can use value from redux-store
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/redux-mixin.html#L14

Example: https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/tickets-block.html#L140